### ISSUE TITLE
Quill inserts a useless line-break before the <ul> element #2905

### ISSUE THREAD

**Issue by donacross** [KillerCodeMonkey/ngx-quill#1639]
Hello, 
Thank you for this great package. 🙏

Today I stumbled on this Quill issue: https://github.com/quilljs/quill/issues/2905

https://user-images.githubusercontent.com/9579634/207529484-7622c796-e36d-44ee-a810-03f7d98cfb2c.mp4

Any suggestion on how to solve it with ngx-quill ?

"quill": "^1.3.7"
"ngx-quill": "^16.2.0"

Thank you

**Comment by KillerCodeMonkey** [comment]
yep and it is a known quilljs issue. since ngx-quill is just a wrapper around it to work better with angular i can nothing do for you here.

try the suggested workarounds in the quill issue, like setting matchVisual-clipboard config to false. or maybe use divs instead of p-tags (check my demo repo there is an example how to do this)

**Comment by donacross** [comment]
Thank you for your quick answer and take the time to maintain this great wrapper. 🙏
